fairroot/base/event/FairTimeStamp.h

```diff
@@ -74,7 +74,7 @@ class FairTimeStamp : public FairMultiLinkedData_Interface
         return out;
     }
 
-    virtual bool operator<(const FairTimeStamp* rValue) const { return GetTimeStamp() < rValue->GetTimeStamp(); }
+    virtual bool operator<(const FairTimeStamp& rValue) const { return GetTimeStamp() < rValue.GetTimeStamp(); }
```

Comment:
1. Please update the copyright year in the header

2. I don't remember what we concluded about making this virtual? Can somebody please comment?

3. This is a breaking change, because it drops an old API. Should we document this in the Chanelog?

4. I don't really understand, why we use the getters instead of accessing the member variables directly?

   ```suggestion
       virtual bool operator<(const FairTimeStamp& rValue) const { return fTimeStamp < rValue.fTimeStamp; }
   ```

5. Should this also compare the timestamp-error, like the `Compare` method above does?

Reply:
I would prefer to keep it `virtual` in case users need to implement their own version of `operator<`.